Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 build = "build.rs"
 
 [dependencies]
-libmacchina = { version = "7.3.1", features = ["version"] }
+libmacchina = { version = "8.0.0", features = ["version"] }
```

Comment:
This should be changed to:

```
libmacchina = { git = "https://github.com/Macchina-CLI/libmacchina", branch = "8.0.0", features = ["version"] }
```

Reply:
Oh sure yeah, wasn't sure you wanted that. I'll update it